packages/asc-ui/src/components/TextArea/TextArea.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useImperativeHandle } from 'react'
+import useActionOnEscape from '../../utils/hooks/useActionOnEscape'
+import useFocusOnRender from '../../utils/hooks/useFocusOnRender'
+import TextAreaStyle from './TextAreaStyle'
+
+export interface TextAreaMethods
+  extends React.TextareaHTMLAttributes<HTMLTextAreaElement> {
+  setTextAreaRef?: (ref: React.RefObject<HTMLTextAreaElement>) => void
+}
+
+export interface TextAreaProps extends TextAreaMethods {
+  keepFocus?: boolean
+  blurOnEscape?: boolean
+  focusOnRender?: boolean
+  error?: string
```

Comment:
Since the `error` field only shows the 'error' state and not an actual message let's change this to a boolean.

Reply:
Done